projects/plugins/jetpack/extensions/blocks/eventbrite/test/fixtures/jetpack__eventbrite__deprecated-1.parsed.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "blockName": "jetpack/eventbrite",
+        "attrs": {
+            "url": "https://www.eventbrite.com.au/e/the-fever-tree-gin-and-tonic-festival-2021-tickets-140737974069",
+            "eventId": 140737974069
+        },
+        "innerBlocks": [],
+        "innerHTML": "\n<a class=\"wp-block-jetpack-eventbrite eventbrite__direct-link\" href=\"https://www.eventbrite.com.au/e/the-fever-tree-gin-and-tonic-festival-2021-tickets-140737974069\">https://www.eventbrite.com.au/e/the-fever-tree-gin-and-tonic-festival-2021-tickets-140737974069</a>\n",
+        "innerContent": [
+            "\n<a class=\"wp-block-jetpack-eventbrite eventbrite__direct-link\" href=\"https://www.eventbrite.com.au/e/the-fever-tree-gin-and-tonic-festival-2021-tickets-140737974069\">https://www.eventbrite.com.au/e/the-fever-tree-gin-and-tonic-festival-2021-tickets-140737974069</a>\n"
+        ]
+    },
+    {
+        "blockName": null,
+        "attrs": {},
+        "innerBlocks": [],
+        "innerHTML": "\n",
+        "innerContent": [
+            "\n"
+        ]
+    }
```

Comment:
This is the result of a blank line at the end of the `.html` fixture file.

Reply:
😱

Reply:
Maybe I have a fix here: https://github.com/Automattic/jetpack/pull/19180